src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(animal) {
+    this.name = animal;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(animal, health) {
+    super(animal, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(predator) {
```

Comment:
why predator?

Reply:
Yeah, you`re right,
Looks not correct in this case, so I have changed it in the correct way.
And I have noticed your solution with a health default value, I have taken your method 🤫